README.md

```diff
@@ -1,3 +1,53 @@
-# Pre Alignment Quality Control
+# ICGC ARGO Pre Alignment QC Workflow
+
+This repository maintains the source code of the ICGC ARGO Pre Alignment QC Workflow for DNA/RNA Seq. The workflow is written in [Nextflow](https://www.nextflow.io/) workflow language using DSLv2, with modules imported from other ICGC ARGO GitHub repositories. Specifically, here are repositories maintaining various of tools/modules:
+
+* https://github.com/icgc-argo-workflows/argo-qc-tools
+* https://github.com/icgc-argo-workflows/dna-seq-processing-tools
+* https://github.com/icgc-argo-workflows/data-processing-utility-tools
+* https://github.com/icgc-argo-workflows/nextflow-dna-seq-processing-tools
+
+Each Nextflow module (including associated container image which is registered in ghcr.io) is strictly
+version controlled and released independently. To ensure reproducibility the pipeline declares explicitly
+which specific version of a module is to be imported.
+
+## Major tasks performed in the workflow
+* download input sequencing metadata/data from `SONG/SCORE`
+* preprocess input sequencing reads (in `FASTQ` or `BAM`) into lane level (aka read group level) `FASTQs`
+* perform `FastQC` analysis for each lane `FASTQs`
+* perform `Cutadapt` analysis for each lane `FASTQs`
+* perform `MultiQC` analysis to generate aggregated results
+* generate `SONG` metadata for all collected QC metrics files and upload them to `SONG/SCORE`
+
+## Inputs
+### Local mode
+- study_id
+- analysis_metadata
+- sequencing_files
+
+### RDPC mode
+- study_id
+- analysis_id
+- song_url
+- score_url
+- api_token
+
+## Outputs
+- file_pair_map: CSV file contains the 3 columns per lane: `read_group_id`, `file_r1`, `file_r2`
```

Comment:
Do you mean per row?

Reply:
Sorry for the confusion. I changed it to
```
- file_pair_map: CSV file with each row contains 3 columns: `read_group_id`, `file_r1`, `file_r2`, which represent information for each lane(aka read group)
```
Is this better? :-)

Reply:
There can be multiple read groups in one lane (dozens to hundreds on a Novaseq)